### ISSUE TITLE
Getting warning for jsx on a regular js file

### ISSUE THREAD

**Issue by FabioAntunes** [ricardofbarros/linter-js-standard#78]
![screenshot from 2016-01-15 10 33 28](https://cloud.githubusercontent.com/assets/2544673/12351220/fb0876a6-bb73-11e5-8545-75f7ac6d0386.png)

On all my es6 files file I'm getting this warning  on the first line, although they have nothing to do with jsx

**Comment by mrakow** [comment]
Warning text (to make this issue easier to find via search):
> Definition for rule 'react/jsx-no-duplicate-props' was not found (react/jsx-no-duplicate-props)at line 1 col 1

Only happens when using standard or semi-standard, not when using happiness. Occurs on all non-empty js files.

Maybe one can copy the happiness rule list over to the other lists.

**Comment by jedthehumanoid** [comment]
This is fixed in semistandard 7.0.3, so all that is needed for linter-js-standard is to update dependency to newer version

Pull request https://github.com/ricardofbarros/linter-js-standard/pull/76 solves this, but it has not been merged yet.

In the meantime, if you want to get rid of this, as a workaround you can change version of semistandard dependency in `.atom/packages/linter-js-standard/package.json`
and do: 

```
npm install
```

in that directory (.atom/packages./linter-js-standard)

**Comment by jedthehumanoid** [comment]
Actually both of the open pull requests solves this.
